lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +30,20 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
     :return: a dictionary with frequencies
     """
 
+    if not isinstance(tokens, list):
+        return None
+    for token in tokens:
+        if not isinstance(token, str):
+            return None
+    all_tokens = len(tokens)
+    calc = {}.fromkeys(tokens, 0)
+    for token in tokens:
+        calc[token] += 1
+    for key in calc:
```

Comment:
Can we avoid the second cycle and create the dictionary needed in one cycle?

Reply:
I've tried to avoid them